modules/GraphStreaming/src/main/resources/org/gephi/graphstreaming/Bundle.properties

```diff
@@ -0,0 +1,8 @@
+OpenIDE-Module-Display-Category=Import
+OpenIDE-Module-Long-Description=\
```

Comment:
This formatting isn't proper and we'll have issue displaying it on the website. Can you please put everything on one line? You can actually use html if you want, example here: https://github.com/mbastian/gephi-plugins/blob/kbrace-filter/modules/KBraceFilter/src/main/resources/org/gephi/plugins/filter/kbrace/Bundle.properties